app/javascript/lib/Entities/Entity.js

```diff
@@ -0,0 +1,11 @@
+export default class Entity {
+    #representation;
```

Comment:
I'm not familiar with this in JS. What does `#representation` do here?

Reply:
Apparently, that's how we declare private modifiers in js

Helps ensure that `representation` is read-only since it's exposed through the getter but has no setter